.kokoro/release/docs.cfg

```diff
@@ -0,0 +1,32 @@
+# service account used to publish up-to-date docs.
+before_action {
+  fetch_keystore {
+    keystore_resource {
+      keystore_config_id: 73713
+      keyname: "docuploader_service_account"
+    }
+  }
+}
+
+# doc publications use a Python image.
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/python"
```

Comment:
I think the node images have Python 3.7 in them: https://github.com/googleapis/testing-infra-docker/blob/master/node/8-user/Dockerfile